doc/manuscript/translocation.qmd

```diff
@@ -674,14 +678,125 @@ Therefore, although some outlier variant associations have a more limited geogra
+For each individual $i=1, ..., M$ on each survey $j=1, ..., n_j$: $o_{i, j} = 1$ if the individual was not detected, and $o_{i, j}=2$ if the individual was detected. 
+Capture histories of $M$ individuals are modeled, though only $N_s$ individuals were captured.
+This parameter expanded data augmentation allows us to capture the possibility that undetected individuals may have recruited into the adult population [@royle2012parameter]. 
+Here, $M$ was chosen to be three times the number of observed individuals ($3N_s$) to be considerably greater than our prior guess of $N_s$. 
+
+We denote the true state of individual $i$ as $u_{i, t}$ for primary period $t = 1,..., n_t$. 
+The four states that we consider are: $u_{i, t} = 1$ for individuals that have not recruited, $u_{i, t} =2$ for live adults, and $u_{i, t} = 3$ for dead adults. 
+Each survey $j=1, ..., n_j$ occurs in one of the $n_t$ primary periods, and we denote the primary period in which survey $j$ takes place as $t_j$. 
+Each primary period $t$ occurs within one year, but within a year there are multiple primary periods.
+We set the year containing the first primary period to $y_{t = 1} = 1$, and generally $y_t$ represents the year containing primary period $t$. 
+Years increment by one until the final year of the mark recapture efforts, which we denote $n_y$: $y \in \{1, 2, ..., n_y\}$.
+We assume that within a primary period, the state of each individual does not change (i.e., individuals do not recruit into the adult population, gain or lose Bd infection, or die). 
+This assumption is justified by the short time intervals between surveys within primary periods.
+
+Live individuals are detected with probability $p_j$, which is modeled as:
+
+$$p_j = \text{logit}^{-1}(X_j^{(p)} \beta^{(p)}),$$
+
+where $X_j^{(p)}$ is a known row vector and $\beta^{(p)}$ an unknown parameter vector. 
+Not recruited and dead individuals are never captured. 
+We bundle these assumptions about the observation probabilities for survey $j$ into an emission matrix $\Omega_j$:
+
+$$
+\Omega_j =
+\begin{blockarray}{ccc}
+  \text{Not detected} & \text{Detected} \\
+\begin{block}{(cc)c}
+  1 & 0 & \text{Not recruited} \\
+  1 - p_j & p_j & \text{Alive} \\
+  1 & 0 & \text{Dead} \\
+\end{block}
+\end{blockarray}
+$$
+
+The state transition matrix $\Psi_{t, i}$ contains the probabilities of individual $i$ transitioning from state $u_{i, t}$ (rows) to $u_{i, t+1}$ (columns) between primary period $t$ and $t+1$. 
+For unintroduced individuals which naturally recruit, this matrix is given by: 
+
+$$
+\Psi_{t, i} =
+\begin{blockarray}{cccc}
+  \text{Not recruited} & \text{Alive} & \text{Dead} \\
+\begin{block}{(ccc)c}
+  1 - \lambda_t & \lambda_t & 0 & \text{Not recruited} \\
+  0 & \phi_t & 1 - \phi_t & \text{Alive} \\
+  0 & 1 & 1 & \text{Dead} \\
+\end{block}
+\end{blockarray},
+$$
+where $\lambda_t$ is the probability of recruiting in time $t$ and $\phi_t$ is the probability of survival in time $t$.
```

Comment:
Either here or in the modeling section, we should say that we used this $\phi_t$ to calculate the survival probability of naturally recruited individuals as shown in Figure S7.